lxd/auth/drivers/tls.go

```diff
@@ -154,10 +154,7 @@ func (t *tls) allowProjectUnspecificEntityType(entitlement auth.Entitlement, ent
 	switch entityType {
 	case entity.TypeServer:
 		// Restricted TLS certificates have the following entitlements on server.
-		return shared.ValueInSlice(entitlement, []auth.Entitlement{auth.EntitlementCanView, auth.EntitlementCanViewResources, auth.EntitlementCanViewMetrics})
-	case entity.TypeStoragePool:
```

Comment:
Should this return an error then?

> We should never expect the authorizer to be called with this entitlement on these entities.

Reply:
We could but the entitlements are now being validated when CheckPermission or GetPermissionChecker is called. So an error will have been returned at that point.